Core/include/Acts/EventData/SingleFreeTrackParameters.hpp

```diff
@@ -11,26 +11,23 @@
 #include <iomanip>
 #include <ostream>
 
-#include "Acts/EventData/detail/coordinate_transformations.hpp"
 #include "Acts/Utilities/Definitions.hpp"
 
 namespace Acts {
 
-/// @class SingleFreeParameters
-///
-/// @brief Container class for free parameters
+/// Free track parameters not bound to a surface.
 ///
 /// This is a base class for neutral and charged free parameters. All parameters
 /// and the corresponding covariance matrix is stored in global coordinates. It
 /// is assumed that the order of the parameters (and thereby the order of the
 /// entries of the covariance as well) is given as (position_x, position_y,
 /// position_z, time, direction_x, direction_y, direction_z, charge /
 /// |momentum|).
-/// @tparam ChargePolicy Parameter that describes if the particle is charged or
-/// neutral
+///
+/// @tparam ChargePolicy Selection typt if the particle is charged or neutral
```

Comment:
typt ? Is it an abbreviation I don't understand or a  typo ?

Reply:
Typo. Thanks for catching that.